DirectProgramming/DPC++FPGA/Tutorials/Features/ac_fixed/README.md

```diff
@@ -0,0 +1,291 @@
+
+In contrast, the kernel `ConstructFromACFixed` in the function `TestConstructFromACFixed` constructs an `ac_fixed` object from an accessor to another `ac_fixed` object. This consumes far less area than the previous kernel. See the section on *Examining the Reports* below to understand where to look for this difference within the optimization reports.
+
+### Using the `ac_fixed` Math Functions
+
+To use the `ac_fixed` math functions in your code, include the following header:
+
+```cpp
+#include <sycl/ext/intel/ac_types/ac_fixed_math.hpp>
+```
+
+The functions `TestCalculateWithFloat` and `TestCalculateWithACFixed` in this tutorial design contain the kernels `CalculateWithFloat` and `CalculateWithACFixed` respectively. Both calculate the simple expression:
+```cpp
+  square_root ( sine(x) * sine(x) + cosine(x) * cosine(x) )
+```
+for some input `x`.
+
+The kernel `CalculateWithFloat` uses floating point values and the standard math library while `CalculateWithACFixed` uses `ac_fixed` values and the `ac_fixed` math library.
+
+In the kernel `CalculateWithACFixed`, the `sin_fixed` and `cos_fixed` functions require the integer part's bit width to be 3, and the input value range to be within [-pi, pi]. Therefore, `ac_fixed` inputs are instantiated with the following parameters:
+
+```cpp
+  W = 10, I = 3, S = true
+```
+
+In this tutorial, the `ac_fixed` numbers are smaller in size than floating point numbers, which results in a reduction of the FPGA resources at the expense of accuracy. To see the trade-offs in accuracy, compare the numeric results of the operations. The area utilization differences will be discussed in the section on *Examining the Reports*.
+
+When you use the `ac_fixed` library, keep the following points in mind:
+
+- Input Bit Width and Input Value Range Limits
+
+    The fixed-point math functions have bit width and input value range requirements. All bit width and input value range requirements are documented at the top of the `ac_fixed_math.hpp` files.
+
+- Return Types
+
+    For fixed-point functions, each function has a default return type. Assigning the result to a non-default return type triggers a type conversion and can cause an increase in logic use or a loss of accuracy in your results.  All return types are documented at the top of the `ac_fixed_math.hpp` file. For example, for `sin_fixed` and `cos_fixed`, the input type is `ac_fixed<W, 3, true>`, and the output type is `ac_fixed<W-1, 2, true>`. You can also use the `auto` type to let the compiler use the default return type.
+
+- Accuracy
+   - Floating point vs Fixed point
+
+     The host program for this tutorial shows the accuracy differences between the correct result and the result provided by the math library functions. The floating point version generates a more accurate result than the fixed point version.
```

Comment:
Please precise that this is the case here because of the fixed-point datatype chosen for this example. One could have chosen a different ac_fixed datatype that would have resulted in a more accurate than floating point result